src/server/index.js

```diff
@@ -1,28 +1,92 @@
+/* eslint-disable react-hooks/rules-of-hooks */
 const {
-  getSchema: storeApiGetSchema,
-  getContextFactory: storeApiGetContextFactory,
-} = require('@faststore/api')
+  envelop,
+  useExtendContext,
+  useMaskedErrors,
+  useSchema,
+} = require('@envelop/core')
+const { useGraphQlJit } = require('@envelop/graphql-jit')
+const { useParserCache } = require('@envelop/parser-cache')
+const { useValidationCache } = require('@envelop/validation-cache')
+const { getContextFactory, getSchema } = require('@faststore/api')
+const { GraphQLError } = require('graphql')
 
-const {
-  platform,
-  channel,
-  api: { storeId, environment },
-} = require('../../store.config')
-
-const options = {
-  platform,
-  account: storeId,
-  environment,
-  channel,
+const persisted = require('../../@generated/graphql/persisted.json')
+const storeConfig = require('../../store.config')
+
+const persistedQueries = new Map(Object.entries(persisted))
+
+const apiOptions = {
+  platform: storeConfig.platform,
+  account: storeConfig.api.storeId,
+  environment: storeConfig.api.environment,
+  channel: storeConfig.channel,
 }
 
-const schema = storeApiGetSchema(options)
-const contextFactory = storeApiGetContextFactory(options)
+const apiSchema = getSchema(apiOptions)
+const apiContextFactory = getContextFactory(apiOptions)
+
+const isBadRequestError = (err) => {
+  return err.originalError && err.originalError.name === 'BadRequestError'
+}
 
-const getSchema = () => schema
-const getContextFactory = () => contextFactory
+const formatError = (err) => {
+  console.error(err)
+
+  if (err instanceof GraphQLError) {
+    if (!isBadRequestError(err)) {
+      return new GraphQLError('Sorry, something went wrong.')
+    }
+
+    return err
+  }
```

Comment:
```suggestion
  if (err instanceof GraphQLError && isBadRequestError(err)) {
    return err
  }
```

Make the code branchless as much as possible.

Reply:
nice!